NewsApp/app/src/main/java/com/example/newsapp/data/room/UserDao.kt

```diff
@@ -9,7 +9,6 @@ import com.example.newsapp.data.entities.User
 @Dao
 interface UserDao {
 
-    //TODO test how the abort works.
     @Insert(onConflict = OnConflictStrategy.ABORT)
```

Comment:
write a comment..it will be aborted in case of duplicate email. I am sure we'll forget that tomorrow 😄